code/lib/source-loader/package.json

```diff
@@ -51,5 +59,10 @@
   "publishConfig": {
     "access": "public"
   },
+  "bundler": {
+    "entries": [
+      "./src/index.ts"
+    ]
```

Comment:
```suggestion
    ],
    "platform": "node"
```

Reply:
Just curious, why was this needed?

Reply:
I see you resolved this without answering, but I'd still love to understand so I can know whether it's needed in the future. Normally I've only added it when there's an error, but should we be adding it for any package that we expect to only run in node? Is that why it's added here?

Reply:
source loader is node only. This ensures that even the ESM output is compiled for executing in node.